hack/update-bazel.sh

```diff
@@ -37,5 +37,13 @@ touch "${KUBE_ROOT}/vendor/BUILD"
 gazelle fix \
     -build_file_name=BUILD,BUILD.bazel \
     -external=vendored \
+    -proto=legacy \
     -mode=fix
+# gazelle gets confused by our staging/ directory, prepending an extra
+# "k8s.io/kubernetes/staging/src" to the import path.
+# gazelle won't follow the symlinks in vendor/, so we can't just exclude
+# staging/. Instead we just fix the bad paths with sed.
+find staging -name BUILD -o -name BUILD.bazel | \
+  xargs sed -i 's|\(importpath = "\)k8s.io/kubernetes/staging/src/\(.*\)|\1\2|'
```

Comment:
Yeah, the sed here isn't pretty, but we have to do something similar already to update staging godeps:
https://github.com/kubernetes/kubernetes/blob/97e002352f459aebf7344babf707837b4f62be2f/hack/update-staging-godeps-dockerized.sh#L79